eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -0,0 +1,5 @@
+<UsageData>
```

Comment:
What is this file for?

Reply:
Please see https://github.com/dotnet/source-build/blob/main/Documentation/planning/arcade-powered-source-build/onboarding/local-onboarding.md#engsourcebuildprebuiltbaselinexml